public/static/js/spawned_hosts.js

```diff
@@ -116,8 +116,8 @@ mciModule.controller('SpawnedHostsCtrl', ['$scope', '$window', '$timeout', 'mciS
       if (!host.isTerminated && new Date(host.expiration_time) > new Date("0001-01-01T00:00:00Z")) {
         if (host.no_expiration) {
           host.expires_in = "never";
-          host.original_expiration = new Date();
-          host.current_expiration = null;
+          host.original_expiration = null;
+          host.current_expiration = new Date();
```

Comment:
I think you didn't want to do this. If the host has no expiration then there is no current expiration.

Reply:
Changed so that it's set to null here